gemrb/plugins/AREImporter/AREImporter.cpp

```diff
@@ -1698,21 +1696,17 @@ int AREImporter::PutHeader(DataStream *stream, const Map *map) const
 	stream->WriteDword(time); //lastsaved
 	stream->WriteDword(map->AreaFlags);
 
-	stream->WriteFilling(8); // northref
-	stream->WriteDword(tmpDword);
-	stream->WriteFilling(8); // westref
-	stream->WriteDword(tmpDword);
-	stream->WriteFilling(8); // southref
-	stream->WriteDword(tmpDword);
-	stream->WriteFilling(8); // eastref
-	stream->WriteDword(tmpDword);
+	stream->WriteFilling(12); // northref
+	stream->WriteFilling(12); // westref
+	stream->WriteFilling(12); // southref
+	stream->WriteFilling(12); // eastref
```

Comment:
We should instead read all these 8 fields and write them back here. They're used in the originals.

Reply:
yeah, thats something to keep in mind for #1404.

Reply:
Exactly. We don't need them for travel, since the world map has all the needed info as well, but it would lower the amount of differences in saves and perhaps allows for simplification of our world map adjacency logic.